Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "osmpbfreader"
-version = "0.16.0"
+version = "0.17.0"
```

As self_cell is an implementation detail not exposed in public API, no need to do a semver break:

```suggestion
version = "0.16.1"
```